redfish/provider/resource_redfish_dell_lifecycle_controller_attributes.go

```diff
@@ -0,0 +1,375 @@
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	service, err := NewConfig(r.p, &plan.RedfishServer)
+	if err != nil {
+		resp.Diagnostics.AddError("service error", err.Error())
+		return
+	}
+
+	diags = updateRedfishDellLCAttributes(ctx, service, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	tflog.Trace(ctx, "resource_DellLCAttributes create: updating state finished, saving ...")
+	// Save into State
+	diags = resp.State.Set(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	tflog.Trace(ctx, "resource_DellLCAttributes create: finish")
+}
+
+// Read refreshes the Terraform state with the latest data.
+func (r *dellLCAttributesResource) Read(ctx context.Context, req resource.ReadRequest, resp *resource.ReadResponse) {
+	tflog.Trace(ctx, "resource_DellLCAttributes read: started")
+	var state models.DellLCAttributes
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	service, err := NewConfig(r.p, &state.RedfishServer)
+	if err != nil {
+		resp.Diagnostics.AddError("service error", err.Error())
+		return
+	}
+
+	diags = readRedfishDellLCAttributes(ctx, service, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	tflog.Trace(ctx, "resource_DellLCAttributes read: finished reading state")
+	// Save into State
+	diags = resp.State.Set(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	tflog.Trace(ctx, "resource_DellLCAttributes read: finished")
+}
+
+// Update updates the resource and sets the updated Terraform state on success.
+func (r *dellLCAttributesResource) Update(ctx context.Context, req resource.UpdateRequest, resp *resource.UpdateResponse) {
+	// Get state Data
+	tflog.Trace(ctx, "resource_DellLCAttributes update: started")
+	var plan models.DellLCAttributes
+
+	// Get plan Data
+	diags := req.Plan.Get(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	service, err := NewConfig(r.p, &plan.RedfishServer)
+	if err != nil {
+		resp.Diagnostics.AddError("service error", err.Error())
+		return
+	}
+
+	diags = updateRedfishDellLCAttributes(ctx, service, &plan)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
+	tflog.Trace(ctx, "resource_DellLCAttributes update: finished state update")
+	// Save into State
+	diags = resp.State.Set(ctx, &plan)
+	resp.Diagnostics.Append(diags...)
+	tflog.Trace(ctx, "resource_DellLCAttributes update: finished")
+}
+
+// Delete deletes the resource and removes the Terraform state on success.
+func (*dellLCAttributesResource) Delete(ctx context.Context, req resource.DeleteRequest, resp *resource.DeleteResponse) {
+	tflog.Trace(ctx, "resource_DellLCAttributes delete: started")
+	// Get State Data
+	var state models.DellLCAttributes
+	diags := req.State.Get(ctx, &state)
+	resp.Diagnostics.Append(diags...)
+	if resp.Diagnostics.HasError() {
+		return
+	}
+
```

Comment:
Looks like redundant, please remove if not required.

Reply:
removed